dpnp/random/dpnp_iface_random.py

```diff
@@ -98,11 +98,18 @@
 ]
 
 
-def _get_random_state():
-    global _dpnp_random_state
-    if _dpnp_random_state is None:
-        _dpnp_random_state = RandomState()
-    return  _dpnp_random_state
+def _get_random_state(device=None, sycl_queue=None):
+    global _dpnp_random_states
+
```

Comment:
```suggestion
    global _dpnp_random_states

    if not isinstance(_dpnp_random_states, dict):
         _dpnp_random_states = dict()
```

Ensure that internal variable can not be reset to an object that does not support `__setitem__` with `SyclQueue` as a key.